src/map/atcommand.cpp

```diff
@@ -4102,10 +4102,10 @@ ATCE atcommand_pvp(Session *s, dumb_ptr<map_session_data> sd,
 
     if (chan < 1) {
         sd->state.pvpchannel = 1;
-        clif_displaymessage(s, "##3PvP : ##BOn"_s);
+        clif_displaymessage(s, "PvP : ##BOn"_s);
```

Comment:
I would rather make it `PvP : ##3##BOn` to not remove the color from the message